packages/aws-cdk/test/integ/cli/test-cdk-deploy-all.sh

```diff
@@ -6,7 +6,7 @@ source ${scriptdir}/common.bash
 
 setup
 
-stack_arns=$(cdk deploy ${STACK_NAME_PREFIX}-test-\*)
+stack_arns=$(cdk deploy ${STACK_NAME_PREFIX}-test-\* -v)
```

Comment:
Is this intentional?

Reply:
Needed it to figure out what's wrong. Not strictly necessary, doesn't harm either. I'll get rid of it.